Automattic/sensei
# Add newsletter step

Fixes https://github.com/Automattic/sensei/issues/5646

Based on https://github.com/Automattic/sensei/pull/5703

### Changes proposed in this Pull Request

* It tweaks the usage step in order to make a part of the styles more generic.
* It introduces the new newsletter step to the onboarding.

The newsletter step will have a new iteration in a new PR after a talk with Andrei about the illustration position. More details: p1663369234771779-slack-C013QUH20TS

### Testing instructions


* Navigate to the usage tracking step (/wp-admin/admin.php?page=sensei_setup_wizard&step=tracking), and make sure the styles continue working properly.
* Navigate to the next step, and make sure you go to the newsletter step.
* Check that the signup button opens the new window to complete the signup, and move to the next onboarding step.
* Also, make sure the Skip button goes to the next step without signing up to the newsletter.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/191114894-b7cb5568-2f60-4af7-aed7-e5820b987049.mov
